IsogeomGenerator/isg.py

```diff
@@ -532,9 +542,12 @@ def __compare_surfs(self, v1, v2, norm):
 
                 if match:
                     # match was found so s1 and s2 are coincident
-                    # delete s2 (all triangles and all vertices that are
-                    # not on the outer curve)
-                    tris2 = self.__get_surf_triangles(verts2.keys())
+                    # delete s2 and remove tris/verts from surf and vol
+                    tris2 = self.mb.get_entities_by_type(s2, types.MBTRI)
+                    self.mb.remove_entities(s2, tris2)
+                    self.mb.remove_entities(v2[1], tris2)
```

Comment:
nonblocking: Is this a volume meshset? Are there triangles in the volume mesh set? (I didn't think DAGMC Volume meshsets had any triangles in them....)

Reply:
After the convos I had with Yuehan and then discovering that my volumes do have triangles in them, I was indeed very confused. I believe it is because the volume files are are read in a new meshset each time and I use that meshset as the volume meshset. So the triangles are all there already. It might make more sense to create a new entity handle for each new volume (instead of using the fileset).